app/components/course-stage-dropdown.hbs

```diff
@@ -0,0 +1,69 @@
+<div data-test-course-stage-dropdown ...attributes>
+  {{! @glint-expect-error: not ts-ified yet }}
+  <BasicDropdown @renderInPlace={{true}} @horizontalPosition="auto-right" as |dd|>
+    <dd.Trigger>
+      <button
+        type="button"
+        class="inline-flex justify-center w-full rounded-md border border-gray-300
+          {{if dd.isOpen 'bg-gray-50' 'bg-white'}}
+          shadow-sm px-4 py-2 text-sm font-medium text-gray-600 hover:bg-gray-50 focus:outline-none"
+        data-test-course-stage-dropdown-trigger
+      >
+        <span class="mr-1" data-test-current-course-stage-name>
+          {{#if this.isAllCourseStagesOptionSelected}}
+            All Stages
+          {{else}}
+            {{@selectedCourseStage.name}}
+          {{/if}}
+        </span>
+
+        {{svg-jar "chevron-down" class=(concat "fill-current w-5 transform transition-all " (if dd.isOpen "text-teal-500" ""))}}
+      </button>
+    </dd.Trigger>
+    <dd.Content>
+      <div class="bg-white rounded-lg shadow-lg border py-2 my-1 w-56" data-test-course-stage-dropdown-content>
+        <div
+          class="flex justify-between items-center py-2 px-4 group hover:bg-gray-50 cursor-pointer"
```

Comment:
What is `group` used for here?

Reply:
Why does this have `justify-between` when there's only one item inside the flex div?